src/main/java/Storage.java

```diff
@@ -0,0 +1,170 @@
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.ToDo;
+
+import java.nio.file.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.io.IOException;
+import java.io.BufferedWriter;
+import java.io.BufferedReader;
+import java.io.FileWriter;
+import java.io.FileReader;
+import java.util.Objects;
+import java.nio.file.Files;
```

Comment:
Perhaps you can consider arranging the import statements in lexicographical order and add line breaks between them appropriately?